src/azul/collections.py

```diff
@@ -97,8 +108,11 @@ def __init__(self, *maps: Mapping[K, V], override: bool = False):
         self.merge(maps)
 
     @classmethod
-    def from_iterable(cls, maps: Iterable[Mapping[K, V]], /) -> Self:
+    def from_iterable(cls, maps: Iterable[Mapping[K, V]],
+                      /,
```

Comment:
This only forces `maps` to be passed as a positional. It doesn't prevent `override` from being passed as a positional.